roles/network_plugin/calico/tasks/install.yml

```diff
@@ -240,7 +240,8 @@
           "logSeverityScreen": "Info",
           {% if not calico_no_global_as_num|default(false) %}"asNumber": {{ global_as_num }},{% endif %}
           "nodeToNodeMeshEnabled": {{ nodeToNodeMeshEnabled|default('true') }} ,
-          {% if calico_advertise_cluster_ips|default(false) %}"serviceClusterIPs": [{"cidr": "{{ kube_service_addresses }}" }],{% endif %}
+          {% if calico_advertise_cluster_ips|default(false) %}
+          "serviceClusterIPs": [{"cidr": "{{ kube_service_addresses }}" } {{ ',{"cidr":"' + kube_service_addresses_ipv6 + '"}' if enable_dual_stack_networks else '' }}],{% endif %}
```

Comment:
According to https://docs.projectcalico.org/networking/advertise-service-ips#advertise-service-cluster-ip-addresses this option seems good.
In addition, this kube_service_addresses_ipv6 is available only if enable_dual_stack_networks is true on Kubespray side.
That also looks good for me because of the following description:
```
# Kubernetes internal network for IPv6 services, unused block of space.
# This is only used if enable_dual_stack_networks is set to true
# This provides 4096 IPv6 IPs
kube_service_addresses_ipv6: fd85:ee78:d8a6:8607::1000/116
```

/lgtm